doc/docs/index.md

```diff
@@ -0,0 +1,12 @@
+# nRF Connect RSSI Viewer
+
+nRF Connect RSSI Viewer is a cross-platform tool that shows dBm per frequency in the 2.4 GHz range. It scans the 2400-2480 MHz range, records signal level on particular frequencies, and presents combined results in several ways.
```

Comment:
“particular frequencies” and “combined results in several ways” seems a bit wishy-washy to me. How about:

```suggestion
nRF Connect RSSI Viewer is a cross-platform tool that shows dBm per frequency in the 2.4 GHz range. It scans the 2400-2480 MHz range, records signal level on the Bluetooth Low Energy channel frequencies, and presents the results.
```